collect_app/src/main/res/values/attrs.xml

```diff
@@ -2,6 +2,7 @@
 <resources>
     <declare-styleable name="Collect">
         <attr name="iconColor" format="color" />
+        <attr name="sliderTrackColor" format="color" />
```

Comment:
I think you misunderstood what I was trying to communicate before. I just meant if we were missing one of the standard material theme colors we might have to add it to our themes. Sorry if that was confusing! 

The theme shouldn't have to know that there is a slider that needs a color right? The slider should use a material theme color we have a value for not a new custom one.

Reply:
Oh now I get it. We have values for almost all the theme colors except for `colorBackground`. However, I feel that I shouldn't be using it as `sliderTrackColor`, as it doesn't sound correct. I feel like I should use `colorOnBackground` as slider track color. Am I correct? I found that we are not using the value of  `colorOnBackground` anywhere in the app. Should I update it?

Reply:
I'd say `colorOnSurface` probably makes sense?